e2e/landing.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { test, expect } from '@playwright/test';
+
+const landingPageElements = {
+  heading: 'landing-header',
+  callToAction: 'landing-big-cta',
+  certifications: 'certifications',
+  curriculumBtns: 'curriculum-map-button',
+  testimonials: 'testimonial-card',
+  landingPageImage: 'landing-page-figure',
+  faq: 'landing-page-faq'
+} as const;
+
+type LandingPageTypes<T> = T[keyof T];
+
+type LandingPageLogs = LandingPageTypes<typeof landingPageElements>;
+
+const superBlocks = [
```

Comment:
**Suggestion:** I feel like this will break if said superblocks are changed. Is there a way to pick these up from the main repo dynamically? Feel free to ignore if there is no possibility here.

Reply:
@hassaanp I was of the impression that the first phase is to vet out Playwright as a tool, comparing it side by side with Cypress. Therefore I followed the exact same approach that was used to test the landing page in Cypress.